src/views/Settings.vue

```diff
@@ -88,14 +88,14 @@ export default defineComponent({
   methods: {
     setEComStore(store: any) {
       if(this.userProfile) {
-        this.store.dispatch('user/setEcomStore', {
-          'eComStore': this.userProfile.stores.find((str: any) => str.productStoreId == store['detail'].value)
-        })
+        const productStoreId = store['detail'].value
+        this.store.dispatch('user/setEcomStore', { 'productStoreId': productStoreId })
       }
     },
     setShopifyConfig(event: any){
-      const shopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value)
-      this.store.dispatch('user/setCurrentShopifyConfig', shopifyConfig);
+      // const currentShopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value);
+      const shopifyConfigId = event.detail.value;
+      this.store.dispatch('user/setCurrentShopifyConfig', { 'shopifyConfigId': shopifyConfigId });
```

Comment:
Why not?
```suggestion
      this.store.dispatch('user/setCurrentShopifyConfig', { 'shopifyConfigId': event.detail.value });
```

Reply:
Implemented the changes, sir, please review.